subprojects/core/src/main/java/org/gradle/api/internal/project/LifecycleAwareProject.java

```diff
@@ -63,6 +77,7 @@ protected Object propertyMissing(String name) {
     @Override
     protected Object methodMissing(String name, Object args) {
         onMutableStateAccess("invokeMethod");
+        // TODO just delegate.invokeMethod?
```

Comment:
❌

Reply:
I want to keep it, to simplify methods implementation in future